networkconfig/local-testnet.go

```diff
@@ -0,0 +1,24 @@
+package networkconfig
+
+import (
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv/protocol/v2/blockchain/beacon"
+	"math/big"
+)
+
+var LocalNetwork = NetworkConfig{
+	Name:                 "local-network",
+	Beacon:               beacon.NewNetworkWithLocalTestNet(spectypes.PraterNetwork, true),
+	Domain:               spectypes.DomainType{0x0, 0x0, 0x4, 0x1},
+	GenesisEpoch:         1,
+	ETH1SyncOffset:       new(big.Int).SetInt64(9203578),
+	RegistryContractAddr: "0xC3CD9A0aE89Fff83b71b58b6512D43F8a41f363D",
```

Comment:
are these constant values used by the local testnet?

Reply:
If they are not used, I think we should use empty string explicitly instead of removal to indicate that we didn't miss it by mistake.

It's not in the scope of this PR, but I think we need to enable the [exhaustruct](https://github.com/GaijinEntertainment/go-exhaustruct) linter in golangci-lint and configure it only for the `networkconfig` directory to make sure we don't miss any values in the configs. @y0sher your thoughts?

Reply:
yeah I agree, again not in the scope of this pr.